packages/storybook-test/stories/link/link.stories.tsx

```diff
@@ -4,6 +4,90 @@ import { Link } from '../../../components-react/link-react/src/css';
+      fill="#30003C"
+    />
+    <path
+      d="M659.54 400.115C650.577 398.321 649.203 396.314 649.203 393.573C649.203 390.619 651.524 389.038 656.272 389.038C661.441 389.038 663.762 391.039 664.929 396.214L677.694 393.573C675.906 384.81 670.731 379.115 656.586 379.115C642.441 379.115 635.591 384.81 635.591 394.42C635.591 402.963 640.233 407.511 653.951 410.465C663.135 412.259 665.136 413.946 665.136 417.007C665.029 420.281 662.602 421.862 657.746 421.862C651.524 421.862 648.143 419.014 646.876 413.526L633.998 416.167C635.371 425.243 640.534 431.999 657.426 431.999C672.513 431.999 678.955 426.084 678.955 416.38C678.955 408.571 675.16 403.402 659.534 400.128L659.54 400.115Z"
+      fill="#30003C"
+    />
+    <path
+      d="M703.444 412.46C703.444 416.474 704.918 418.475 709.246 418.475C711.467 418.475 713.894 417.841 715.795 417.208L717.376 430.079C713.574 431.24 711.04 431.98 705.552 431.98C693.947 431.98 689.092 424.283 689.092 414.78V391.879H682.969V380.482H689.092V367.611H703.444V380.482H716.629V391.879H703.444V412.46Z"
+      fill="#30003C"
+    />
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M747.768 379.109C731.519 379.111 722.124 389.241 722.124 406.545C722.124 423.851 732.468 431.873 747.554 431.873C761.8 431.873 767.608 425.331 769.823 416.675L757.792 413.4C756.524 417.202 754.517 420.369 747.655 420.369C740.792 420.369 737.104 416.781 736.263 408.759H770.136C770.35 406.118 770.45 404.117 770.45 402.01C770.45 388.187 763.176 379.111 747.768 379.109ZM736.37 401.483C737.104 393.253 740.905 389.872 747.347 389.872C753.789 389.872 757.058 393.673 757.265 401.483H736.37Z"
+      fill="#30003C"
+    />
+    <path
+      d="M747.768 379.109C747.77 379.109 747.772 379.109 747.774 379.109H747.761C747.763 379.109 747.766 379.109 747.768 379.109Z"
+      fill="#30003C"
+    />
+    <path
+      d="M825.224 394.1C827.978 384.283 833.674 379.108 841.91 379.108L841.904 379.096V379.077C851.714 379.077 856.676 385.726 856.676 396.596V430.474H842.224V400.611C842.224 395.549 839.909 392.068 834.621 392.068C828.499 392.068 824.704 396.396 824.704 402.624V430.487H810.351V400.623C810.351 395.561 807.924 392.08 802.749 392.08C796.633 392.08 792.725 396.408 792.725 402.637V430.499H778.373V380.476H792.405V393.88H793.459C796.307 384.177 802.328 379.108 810.239 379.108C819.102 379.108 823.43 384.71 824.064 394.1H825.224Z"
+      fill="#30003C"
+    />
+  </svg>
+);
 
 const meta = {
```

Comment:
Ik denk dat het handig is om al deze link stories in de ExampleBodyText decorator en een Paragraph decorator te zetten.